site/assets/styles/scss/styles.scss

```diff
@@ -123,13 +123,18 @@ img.avatar {
         height: auto;
     }
     .month-nav-wrapper{
-        @media (min-width: 992px){
+        @media (min-width: 576px){
             width: 14rem;
```

Comment:
When we get to longer month names, like `| < December | October > |`, a width of 14rem isn't wide enough. It looks like changing width here to `fit-content` will fix it for all months.